modules/src/ics07_tendermint/client_state.rs

```diff
@@ -0,0 +1,82 @@
+use crate::ics02_client::client_type::ClientType;
+use crate::ics23_commitment::CommitmentRoot;
+
+use crate::ics07_tendermint::header::Header;
+use crate::ics24_host::client::ClientId;
+use serde_derive::{Deserialize, Serialize};
+use std::time::Duration;
+use tendermint::lite::Header as liteHeader;
+
+#[derive(Clone, Debug, PartialEq, Serialize, Deserialize)]
+pub struct ClientState {
+    id: String,
+    trusting_period: Duration,
+    unbonding_period: Duration,
+    latest_header: Header,
```

Comment:
Note that the Header here contains a `SignedHeader` which in turn contains a `Commit`. Looking at the `Commit` struct in the iqlusion/relayer (on master) shows that they are already using tendermint `v0.33.2`which modified the `Commit` struct:
```go
type Commit struct {
	// NOTE: The signatures are in order of address to preserve the bonded
	// ValidatorSet order.
	// Any peer with a block can gossip signatures by index with a peer without
	// recalculating the active ValidatorSet.
	Height     int64       `json:"height"`
	Round      int         `json:"round"`
	BlockID    BlockID     `json:"block_id"`
	Signatures []CommitSig `json:"signatures"`
        // ...
}
```
while this branch uses the v0.32 version of the commit:
```rust
pub struct Commit {
    /// Block ID of the last commit
    pub block_id: block::Id,

    /// Precommits
    pub precommits: Precommits,
}
```
Or in golang: https://github.com/tendermint/tendermint/blob/64e61365c16f74d56ccaa9eb784e05df5b01764c/types/block.go#L489-L494

v0.33 tendermint-rs is currently worked on in https://github.com/informalsystems/tendermint-rs/pull/196

Does it make sense to assume we want to decode v0.33 commits directly (for the sake of #45)? @ancazamfir

Reply:
yes, let's look at 0.33! thx!